Example/Firebase.xcodeproj/project.pbxproj

```diff
@@ -4887,6 +4887,10 @@
 				INFOPLIST_FILE = "$(SRCROOT)/Core/App/macOS/Core-Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks";
 				MTL_ENABLE_DEBUG_INFO = YES;
+				OTHER_LDFLAGS = (
+					"$(inherited)",
+					"-ObjC",
```

Comment:
Was this missed before, or why does it need to be added now?

Reply:
The new pod structure exposed macOS test failures from categories not being linked in. There's no change to iOS since their tests are linked with -all_load.